client/src/pages/Dashboard/Dashboard.tsx

```diff
@@ -13,6 +13,7 @@ import Profile from '../Profile/Profile';
 import Submission from '../Submission/Submission';
 import Contest from '../Contest/Contest';
 import EditProfile from '../Profile/EditProfile/EditProfile';
+import ViewContestSubmissions from '../ViewContestSubmissions/ViewContestSubmissions';
```

Comment:
Can you add a protected route for this page and a way to navigate to it? Maybe via the profile dropdown menu

Reply:
Added a button for now